cmd/glasskube/cmd/update.go

```diff
@@ -117,9 +119,54 @@ func completeInstalledPackageNames(
 	return
 }
 
+func completeUpgradablePackageVersions(
+	cmd *cobra.Command,
+	args []string,
+	toComplete string,
+) ([]string, cobra.ShellCompDirective) {
+
+	var dir cobra.ShellCompDirective
+	config, _, err := kubeconfig.New(config.Kubeconfig)
+	if err != nil {
+		dir = cobra.ShellCompDirectiveError
```

Comment:
Multiple directives can be combined, you already do this at line 152 (see below). I think this should be done everywhere consistently.